### PR TITLE
Add/update DCO, CoC, contributing ... files

### PR DESCRIPTION
**Pull request by mmlb**
## Description

Update non code files to be uniformish amongst all repos.

## Why is this needed

Ensures our repos are all licensed correctly and contain similar/minimum
contribution info files.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Please submit change requests and / or features via [Issues](https://github.com/tinkerbell/terraform-provider-tinkerbell/issues).
+There's no guarantee it'll be changed, but you never know until you try.
+We'll try to add comments as soon as possible, though.
+
+### How to Report a Bug
+
+Bugs are problems in code, in the functionality of an application or in its UI design; you can submit them through [Issues](https://github.com/tinkerbell/terraform-provider-tinkerbell/issues).
+
+## Code Style Guides
```
Comment (invidian) [inline CONTRIBUTING.md:32]: Should something be mentioned here? At least that there is no style guides at the moment or link to something general like https://github.com/golang/go/wiki/CodeReviewComments ?

**Review by invidian — APPROVED** [review]
LGTM, just one comment. Someone feel free to merge, I don't know how those files look in other repos, so can't say for sure this is right.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mmlb) [reply CONTRIBUTING.md:32]: > LGTM, just one comment. Someone feel free to merge, I don't know how those files look in other repos, so can't say for sure this is right.

I'm doing similar PRs to all the other repos, :D

Comment (mmlb) [reply CONTRIBUTING.md:32]: > Should something be mentioned here? At least that there is no style guides at the moment or link to something general like [golang/go/wiki/CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments) ?

Lets leave blank for this PR and then we can follow up with *some* guide lines, but also this maybe repo specific so I'd like the "common stuff" pr first.

**Pull request merged**